deploy/src/loadEnv.js

```diff
@@ -34,6 +37,36 @@ const validateRewardableAddresses = (validators, rewards) => {
   }
 }
 
+function checkValidators(validators, requiredValidators) {
+  if (validators.split(' ').length < requiredValidators) {
+    throw new Error(
+      `The number of validators VALIDATORS = ${
+        validators.split(' ').length
+      } should be bigger or equal the number of required signatures REQUIRED_NUMBER_OF_VALIDATORS = ${requiredValidators} .`
+    )
+  }
+}
+
+function checkGasPrices(gasPrice, prefix) {
+  if (gasPrice.isZero()) {
+    throw new Error(`${prefix}_GAS_PRICE should be bigger than zero.`)
```

Comment:
in fact the gas price could be zero e.g. for `erc-to-native` bridge when the validators are whitelisted so can send transactions with zero gas price.

Reply:
Thanks. Now gas price check is applied for all cases except for home gas price on `erc_to_native` mode https://github.com/poanetwork/poa-bridge-contracts/pull/228/commits/8d35e1e6dd82456a4ce60917abe4b510a016bc9f

Reply:
I think that for future POSDAO chain (as home) it can be zero in the `erc-to-erc` mode as well. What if we just check that the gas price either `0` or more than `100` mWei for any side or any mode? Do you see any issue?

Reply:
The only issue I see is that all home and foreign contracts, except home `HomeBridgeErcToNative`, check that gasPrice value is > `0`. For example: 
https://github.com/poanetwork/poa-bridge-contracts/blob/df1fa675138a0d18f3a276cb89b166c77b9c2482/contracts/upgradeable_contracts/erc20_to_erc20/HomeBridgeErcToErc.sol#L134

So a user will be able to pass the parameters check by setting home or foreign gas price with a value of `0` but then get the initialization transaction of the bridge reverted

Reply:
Got you! Let's keep the checks as you implemented so far. I will open an issue to allow configure 0 gas price for the `erc-to-erc` mode.